libpod/runtime_img.go

```diff
@@ -124,7 +126,74 @@ func (r *Runtime) RemoveImage(image *image.Image, force bool) (string, error) {
 		return "", fmt.Errorf("unable to delete %s (must force) - image is referred to in multiple tags", image.ID())
 	}
 
-	return image.ID(), image.Remove(force)
+	err = image.Remove(force)
+	if err != nil && errors.Cause(err) == storage.ErrImageUsedByContainer {
+		if errors.Cause(err) == storage.ErrImageUsedByContainer {
+			storageImage, err2 := r.getStorageImage(image.ID())
+			if err2 == nil {
+				errStorage := rmStorageContainers(force, storageImage, image.GetStore())
+				if errStorage == nil {
+					err = image.Remove(force)
+				} else {
+					fmt.Printf("A container created with Buildah may be associated with this image.\n")
```

Comment:
No printing in libpod - return this as an error

Reply:
will do....